nextstrain/cli/runner/ambient.py

```diff
@@ -1,5 +1,31 @@
 """
-Run commands in the ambient environment, outside of any container image.
+Run commands in the ambient environment, outside of any container image or managed environment.
+
+The "ambient" runtime allows you to use the Nextstrain CLI with your own ambient
+setup, for when you cannot or do not want to have Nextstrain CLI manage its own
+runtime.
+
+.. versionadded:: 1.5.0
+.. versionchanged:: 5.0.0
+    Renamed from "native" to "ambient".
+
+
+.. _ambient-setup:
+
+Setup
+=====
+
+You will need to make sure all of the Nextstrain software dependencies
+are available locally or "ambiently" on your computer. A common way to do this
+is by manually using `Conda <https://docs.conda.io/en/latest/miniconda.html>`__
```

Comment:
Suggestion: add a link to the general installation page which provides example commands to set up a Conda environment for the ambient runtime.

This would be analogous to the hint in the Docker runtime setup section.

Reply:
Done in acf3453.